src/store/page/page-element.ts

```diff
@@ -1,5 +1,7 @@
+import * as deepAssign from 'deep-assign';
 import { JsonArray, JsonObject, JsonValue } from '../json';
 import * as MobX from 'mobx';
+import * as objectPath from 'object-path';
```

Comment:
Can you please change the case to ObjectPath? We have upper-cased all tool-like imports so far (at least in the store).

Reply:
done